sdk/storage/azblob/zt_examples_test.go

```diff
@@ -201,182 +183,175 @@ func ExampleStorageError() {
 }
 
 // This example shows how to create and use an Azure Storage account Shared Access Signature (SAS).
 func ExampleAccountSASSignatureValues_Sign() {
-	accountName, accountKey := accountInfo()
+	accountName, accountKey := os.Getenv("AZURE_STORAGE_ACCOUNT_NAME"), os.Getenv("AZURE_STORAGE_ACCOUNT_KEY")
 
-	credential, err := NewSharedKeyCredential(accountName, accountKey)
+	credential, err := azblob.NewSharedKeyCredential(accountName, accountKey)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	sasQueryParams, err := AccountSASSignatureValues{
-		Protocol:      SASProtocolHTTPS,
+	sasQueryParams, err := azblob.AccountSASSignatureValues{
+		Protocol:      azblob.SASProtocolHTTPS,
 		ExpiryTime:    time.Now().UTC().Add(48 * time.Hour),
-		Permissions:   AccountSASPermissions{Read: true, List: true}.String(),
-		Services:      AccountSASServices{Blob: true}.String(),
-		ResourceTypes: AccountSASResourceTypes{Container: true, Object: true}.String(),
+		Permissions:   azblob.AccountSASPermissions{Read: true, List: true}.String(),
+		Services:      azblob.AccountSASServices{Blob: true}.String(),
+		ResourceTypes: azblob.AccountSASResourceTypes{Container: true, Object: true}.String(),
 	}.Sign(credential)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	qp := sasQueryParams.Encode()
-	urlToSend := fmt.Sprintf("https://%s.blob.core.windows.net/?%s", accountName, qp)
-	// You can hand off this URL to someone else via any mechanism you choose.
-
-	// ******************************************
+	queryParams := sasQueryParams.Encode()
+	sasURL := fmt.Sprintf("https://%s.blob.core.windows.net/?%s", accountName, queryParams)
 
-	// When someone receives the URL, they can access the resource using it in code like this, or a tool of some variety.
-	serviceClient, err := NewServiceClientWithNoCredential(urlToSend, nil)
+	// This URL can be used to authenticate requests now
+	serviceClient, err := azblob.NewServiceClientWithNoCredential(sasURL, nil)
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	// You can also break a blob URL up into it's constituent parts
-	blobURLParts := NewBlobURLParts(serviceClient.URL())
+	blobURLParts := azblob.NewBlobURLParts(serviceClient.URL())
 	fmt.Printf("SAS expiry time = %s\n", blobURLParts.SAS.ExpiryTime())
 }
 
 // This example demonstrates how to create and use a Blob service Shared Access Signature (SAS)
 func ExampleBlobSASSignatureValues() {
-	// Gather your account key and name from the Azure portal
-	// Supplying them via environment variables is recommended.
-	accountName, accountKey := accountInfo()
+	accountName, accountKey := os.Getenv("AZURE_STORAGE_ACCOUNT_NAME"), os.Getenv("AZURE_STORAGE_ACCOUNT_KEY")
 
-	// Use your storage account's name and key to form a credential object
-	credential, err := NewSharedKeyCredential(accountName, accountKey)
+	credential, err := azblob.NewSharedKeyCredential(accountName, accountKey)
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	containerName := "myContainer"
```

Comment:
Just noticed this one as well.
```suggestion
	containerName := "mycontainer"
```